refactor-me/refactor-me.csproj

```diff
@@ -0,0 +1,151 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Consider enabling SSL by default on local server at development time. This ensures development is close as possible to a production-like environment.